index.html

```diff
@@ -12,6 +12,7 @@
   <link rel="stylesheet"
     href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="description" content="RS School is free-of-charge and community-based education program conducted by The Rolling Scopes developer community since 2013." />
```

Comment:
@dzmitry-varabei, could you review the following copy and let us know if it looks good to you?

Reply:
@MielomankA Please use the following text "RS School offers free, community-driven education courses run by The Rolling Scopes developer community since 2013. Enhance your web development, JavaScript, and front-end skills with us."